textproc/lilypond-devel/Portfile

```diff
@@ -30,32 +30,31 @@ checksums           rmd160  6250667c2a2799ec404e10e3e141101d043ab7f6 \
                     size    17996428
 
 depends_build       port:autoconf \
```

Comment:
Can you use append here ? Its general better to avoid removing deps from portgroups etc.

Reply:
Done.